### ISSUE TITLE
Enhance ERA001 commented out code analysis

### ISSUE THREAD

**Issue by mikeleppane** [charliermarsh/ruff#3993]
Would it be possible to detect commented out code in case you have used triple-quoted strings? It probably won't be trivial thing to do because how to separate from an actual doc string. 

Ruff detects this: 
```python
# print("commented out code")
``` 

but not this:  
```python
""" print("commented out code") """
``` 

Ruff version:
```shell 
ruff 0.0.261
``` 

Thanks for the Ruff! We recently switched from flake8 to ruff for our production code.

**Comment by charliermarsh** [comment]
Thank you for the kind words :) I think I understand where the request is coming from, but I'd rather keep the rule as-is for now -- it seems less common to temporarily disable code by stringifying it, which is the mistake that this rule is intended to catch, and we'd be more likely to flag false positives here as you suggested.
